es-app/src/guis/GuiRandomCollectionOptions.cpp

```diff
@@ -41,21 +42,21 @@ void GuiRandomCollectionOptions::initializeMenu()
 	}
 
 	mMenu.addWithLabel("EXCLUDE GAMES FROM", exclusionCollection);
-	
 
 	// Add option to trim random collection items
-	trimRandom = std::make_shared< OptionListComponent<std::string> >(mWindow, "MAX GAMES", false);
-	
+	trimRandom = std::make_shared<NumberList>(mWindow, "MAX GAMES", false);
+
 	// Add default entry
-	trimRandom->add("ALL", "", Settings::getInstance()->getString("RandomCollectionMaxItems") == "");
-	
-	// add all enabled Custom Systems
+	int maxItems = Settings::getInstance()->getInt("RandomCollectionMaxItems");
```

Comment:
For consistency's sake then, this should be maxGames :)

Reply:
Right. Your previous suggestion led us to make this "maxGames", so we could also change the property name, as there's likely few adopters so far. Or just leave the property name and variable name then, you're right.

Your call - happy to keep as is, now that you reference the property name.